docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -167,10 +162,10 @@ NOTE: `Where` will only filter by existing properties right before deletion.
 [[combining-operators]]
 === Combining operators
 
-All above-mentioned operators can be combined using the `AND`/`OR` operators. 
+All above-mentioned operators can be combined using the `AND`/`OR`/`NOT` operators. 
 They accept an array argument with items of the same format as the `where` argument, which means they can also be nested to form complex combinations.
 
-Say we are picky fans of comedy movies and we only accept ratings below 7 for movies released before the 2000's. 
+Say we dislike comedy movies and we only accept ratings below 7 for movies released before the 2000's. 
 As an exception we also like the movie "The Matrix". However, we do not like any of its sequels. 
 We could subscribe to any updates that we are interested in as follows:
```

Comment:
The wording of this contrived example doesn't really make sense anymore with these changes (the exception from before of The Matrix being a non-comedy movie is no longer relevant)

Reply:
Good catch here!